src/library/browse/browsetablemodel.cpp

```diff
@@ -413,10 +418,10 @@ bool BrowseTableModel::isColumnSortable(int column) {
 }
 
 QAbstractItemDelegate* BrowseTableModel::delegateForColumn(const int i, QObject* pParent) {
-    QTableView* pTableView = qobject_cast<QTableView*>(pParent);
+    WLibraryTableView* pTableView = qobject_cast<WLibraryTableView*>(pParent);
```

Comment:
Minor: This is a typical situation where redundant types can be avoided by using `auto`